workshops/management/commands/check_certificates.py

```diff
@@ -46,13 +50,13 @@ def get_file_people(self, path):
                     for e in os.listdir(path)
                     if e.endswith('.pdf')])
 
-    def report_missing(self, title, kind, items):
-        '''Report missing items.'''
-        if items:
-            print('{0} {1}'.format(kind, title))
-            for i in sorted(list(items)):
-                try:
-                    p = Person.objects.get(username=i)
-                    print(' {0}: {1}'.format(i, p))
-                except Person.DoesNotExist:
-                    print(' {0}'.format(i))
+    def missing(self, report, title, kind, usernames, records):
+        '''Report missing usernames.'''
+        for u in usernames:
+            try:
+                p = Person.objects.get(username=u)
```

Comment:
I find that long variable names that different only in the last character are hard to read, and that short names are OK for iteration variables.